CRM/Volunteer/Form/VolunteerSignUp.php

```diff
@@ -706,4 +768,20 @@ protected function setDestination() {
     $this->_destination = CRM_Utils_System::url($path, $query, FALSE, $fragment);
   }
 
+  /**
+   * If the form preprocessing uncovers problems with the passed needs, display
```

Comment:
Don't comment with business logic.
"Sub-routine of buildQuickForm to display validation messages to the user."